src/Ingridient.js

```diff
@@ -0,0 +1,9 @@
+class Ingridient {
```

Comment:
I'm curious to see how we use this. Will we end up adding more properties? Once we get deeper into the logic, will we simply access from the ingredient data file instead of the class? I have no idea at this point in my coding experience, but I am excited to find out!